engine/src/test/java/org/camunda/bpm/engine/test/history/JobEntityAndJobLogBatchIdTest.java

```diff
@@ -83,14 +85,6 @@ public void init() {
     externalTaskService = engineRule.getExternalTaskService();
   }
 
-  private BpmnModelInstance getUserTaskProcess() {
```

Comment:
❓ Is moving 

- the creation of the `process with the userTask` to `ExecutableProcessUtil` and 
- making it `static` 

an optimisation to make the creation only once or 
is there another reasoning behind this change?

Reply:
Only to avoid code duplication as this is used by different test classes and I thought I'd centralize it. No strong opinion on this though.